src/core/utils/utils.ts

```
@@ -85,3 +85,12 @@ export function getDefaultAccountNetworkType(): NetworkType {
     if (!accounts[0]) return defaultNetworkConfig.DEFAULT_NETWORK_TYPE
     return accounts[0].networkType
 }
+
+export function getNetworkTypeToLocalString(networkValue): string {
+    for (const key in NetworkType) {
```

please improve this, this is doing a loop where its not needed, have an internal mapping rather than looping over several NetworkTypes *and* also throw an error (correct error handling) rather than returning empty value.